timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -521,6 +521,13 @@ module.exports = [
             keywords: ["migrate", "data", "TimescaleDB"],
             excerpt: "Migrate data from a different PostgreSQL database",
           },
+          {
```

Comment:
@jacobprall could I get your help here? This doesn't show up on the navigation when I build the site locally, and I'm not sure what I'm doing wrong. Thanks! 🙂